bin/templatemap_generator.php

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
```

Comment:
@weierophinney 

Here you go: https://github.com/zendframework/zend-view/pull/65

I hope everything is ok, since I am sitting in a train home and don't have access to my main note book. I only have a windows machine here right now.

I guess this PR can be closed if you accept the one at zend-view